pepper-apis/src/test/java/org/broadinstitute/ddp/route/ServerInSameProcessIntegrationTestSuite.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.ddp.route;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * These tests need to have server running in same process. In CircleCi running tests these are treated
+ * differently then other route tests that just need access to the server via http and and the database
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        GetParticipantInfoRouteTest.class,
+        ListCancersRouteTest.class,
+        GetDsmDrugSuggestionsRouteTest.class
+})
+public class ServerInSameProcessIntegrationTestSuite extends IntegrationTestSuite {
+}
```

Comment:
These are the special test that are run separate from all the others in CircleCi as the comment indicates.
Kind of hacky. Tried hard to find a way with JUnit to tag just these tests or add them dynamically but could not find a good solution. Open to suggestions on how to achieve same thing more cleanly.